src/components/v2/Toggle/styles.ts

```diff
@@ -1,8 +1,9 @@
+import { css } from '@emotion/react';
 import { useTheme } from '@mui/material';
 
 export const useStyles = () => {
   const theme = useTheme();
-  return {
+  return css({
```

Comment:
Rather then wrap this in css we should rewrite it to be an interpolated string, that way it is lintable and more readable

Reply:
I have done this here - https://github.com/VenusProtocol/venus-protocol-interface/pull/265

Reply:
I've approved and merged your PR